scripts/prepublish

```diff
@@ -5,6 +5,8 @@ slash=/
 README="$(cat README.md)"
 README="${README//${PREVIOUS_VERSION}${slash}ramda.min.js/${VERSION}${slash}ramda.min.js}"
 README="${README//${PREVIOUS_VERSION%.*}${slash}ramda.min.js/${VERSION%.*}${slash}ramda.min.js}"
+README="${README//${PREVIOUS_VERSION}${slash}dist${slash}ramda.min.js/${VERSION}${slash}dist${slash}ramda.min.js}"
+README="${README//${PREVIOUS_VERSION%.*}${slash}dist${slash}ramda.min.js/${VERSION%.*}${slash}dist${slash}ramda.min.js}"
```

Comment:
Should these lines replace the preceding two lines, or do we really need to perform four substitutions?

Reply:
Currently we need three, the two new lines but only one of the old lines.

Reply:
Okay. Let's remove whichever substitution is no longer necessary.